sirepo/job_driver/__init__.py

```diff
@@ -273,7 +274,7 @@ async def _agent_start(self, op):
                 # POSIT: Canceled errors aren't smothered by any of the below calls
                 await self.kill()
                 await self._do_agent_start(op)
-            except Exception as e:
+            except (Exception, *sirepo.util.ASYNC_CANCELED_ERROR) as e:
```

Comment:
I read about `except*`. This nested/multi-exception thing seems very complicated. Do we need it here? It seems like these are two unrelated exceptions.

Reply:
2f7887c01